operators/KukisGlobal/metadata.json

```diff
@@ -0,0 +1,7 @@
+{
+  "name": "KukisGlobal",
+  "website": "https://www.kukis-global.com",
+  "description": "Kukis Global is one of the top Web3 infrastructure providers and a Lido node operator. We are validating around 1% of the entire Ethereum network with a perfect track record.",
+  "logo": "https://github.com/kukis-global/metadata/blob/main/kg-logo.png?raw=true",
```

Comment:
Hello  @andrewpaulicek - this should be a the raw URL. please check the example and fix it. thanks!

Reply:
Thank you for the review! 
Fix submitted.